test/testutil.py

```diff
@@ -77,3 +80,27 @@ def can_start_rootful_containers():
             return res.stdout.strip() == "true"
         case unknown:
             raise ValueError(f"unknown platform {unknown}")
+
+
+def write_aws_creds(path):
+    key_id = os.environ.get("AWS_ACCESS_KEY_ID")
+    secret_key = os.environ.get("AWS_SECRET_ACCESS_KEY")
+    if not key_id or not secret_key:
+        print("WARNING: aws credentials not available")
+    with open(path, mode="w", encoding="utf-8") as creds_file:
+        creds_file.write("[default]\n")
+        creds_file.write(f"aws_access_key_id = {key_id}\n")
+        creds_file.write(f"aws_secret_access_key = {secret_key}\n")
+
+
+def delete_ami(ami_id):
```

Comment:
(super nitpick) maybe `deregister_ami()` to follow upstream conventions?

Reply:
Done.